Makefile

```diff
@@ -0,0 +1,19 @@
+GRUNT_DOCKER_NAME:=galaxy-client-builder
```

Comment:
Lets call it ``galaxy/client-builder:16.01`` instead, is that okay?

This will match the testing Docker image and reflects that we should put it on dockerhub and version tag it.